include/serial.h

```diff
@@ -114,6 +114,9 @@
 #define SER_ERR_INSTALLED       0x0A    /* A driver is already installed */
 #define SER_ERR_NOT_OPEN        0x0B    /* Driver is not open */
 
+#define SER_A2GS_CHANNEL_A      0x01    /* Apple IIgs printer port */
+#define SER_A2GS_CHANNEL_B      0x00    /* Apple IIgs modem port */
+
```

Comment:
Please remove this again.

Reply:
Where do you I should put them? apple2.h? Or just not put them at all and document the values in doc/?

Reply:
The latter, see my by now added comment for the doc...